vrf_fun/tests/codec.rs

```diff
@@ -1,3 +1,5 @@
+#![cfg(feature = "bincode")]
```

Comment:
Redundant? We already feature gate the tests in this file

Reply:
no because of warnings. Nothing in this file does anything without that feature so it should be excluded without it.